protocol/tx/spend.go

```diff
@@ -9,17 +9,43 @@ type spend struct {
 		ExtHash     bc.Hash
 	}
 	ordinal int
+
+	// SpentOutput contains (a pointer to) the manifested entry
+	// corresponding to body.SpentOutput. It may be nil, in which case
+	// spentInfo must be non-nil.
+	SpentOutput *output
+
+	// spentInfo contains validation-essential data in case the spent
+	// output object is not present (in SpentOutput).
```

Comment:
Resolution: yes it's desirable and is being addressed as part of https://github.com/chain/chain/issues/648